File: an2_sem4/MPP/Homework/sapt78/cs/FestivalSellpoint/FestivalSellpoint.Service/IBiletService.cs
using FestivalSellpoint.Domain;
using System.Collections.Generic;

namespace FestivalSellpoint.Service
{
    public interface IBiletService : IService<int, Bilet>
    {
        IEnumerable<Bilet> GetBySpectacol(Spectacol spectacol);
    }
}

File: an2_sem4/MPP/Homework/sapt14/service/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/service/IBiletService.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.service;

import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain.Bilet;
import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain.Spectacol;
import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.repo.EntityRepoException;

public interface IBiletService extends IService<Integer, Bilet> {
    Iterable<Bilet> getBySpectacol(Spectacol spectacol) throws EntityRepoException;
}
